dev-docs/bidders/appnexus.md

```diff
@@ -43,4 +53,42 @@ var adUnits = [{
 }]
 {% endhighlight %}
 
-(Sizes set in `adUnit` object will also apply to the AppNexus bid requests.)
+{: .alert.alert-info :}
+Sizes set in the `adUnit` object will also apply to the AppNexus bid requests.
+
+<a name="appnexus-aspect-ratios" />
+
+#### Specify aspect ratios for native images and icons
```

Comment:
Is native aspect ratio support documented somewhere else that is not specific to the appnexus adapter?

I think this is confusing, because I don't see any mention of "absolute sizes" for native images/ icons which "appnexus" can also support.